kernels/volk/volk_32u_reverse_32u.h

```diff
@@ -290,8 +292,15 @@ volk_32u_reverse_32u_neonv8(uint32_t* out, const uint32_t* in, unsigned int num_
 
 #ifdef LV_HAVE_NEON
 #include <arm_neon.h>
-
-#if defined(__aarch64__)
+#ifdef _MSC_VER
+#define DO_RBIT                                                                 \
+    *out_ptr = _byteswap_ulong(*in_ptr);                                        \
+    *out_ptr = ((*out_ptr & 0x55555555) << 1) | ((*out_ptr & 0xAAAAAAAA) >> 1); \
+    *out_ptr = ((*out_ptr & 0x33333333) << 2) | ((*out_ptr & 0xCCCCCCCC) >> 2); \
+    *out_ptr = ((*out_ptr & 0x0F0F0F0F) << 4) | ((*out_ptr & 0xF0F0F0F0) >> 4); \
+    in_ptr++;                                                                   \
+    out_ptr++;
+#elif defined(__aarch64__)
```

Comment:
This section indicates that we already had special treatment for some conditions. Now, it looks like we exchange them for another set of conditions. This change would warrant a comment on the specifics of different platforms. This will help anyone looking at this in the future.